patchwork-events-world/src/main/java/net/minecraftforge/event/world/SaplingGrowTreeEvent.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.world;
+
+import java.util.Random;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IWorld;
+
+/**
+ * SaplingGrowTreeEvent is fired when a sapling grows into a tree.<br>
+ * This event is fired during sapling growth in
+ * {@link net.minecraft.block.SaplingBlock#generate(IWorld, BlockPos, BlockState, Random)}.<br>
+ * <br>
+ * {@link #pos} contains the coordinates of the growing sapling. <br>
+ * {@link #rand} contains an instance of Random for use. <br>
+ * <br>
+ * This event is not cancellable.<br>
+ * <br>
+ * This event has a result.<br>
+ * This result determines if the sapling is allowed to grow. <br>
```

Comment:
IIRC generally we turn Forge's use of linebreaks into proper paragraphs.

Reply:
Kit is correct.